### DISCUSSION TITLE
Server does not restart.

### DISCUSSION THREAD

**Discussion by Alphanumerisch** [amidaware/tacticalrmm#2015] [General]
TRMM Version: v0.19.1

Hey, hello everyone.

I've been running TRMM for several months now.
I think it's a really good program. Thanks for that.

I have set up everything as described in the documentation.

Unfortunately, my servers are still in “Restart required” status

The status simply does not change.

Why don't the servers restart after updates?

**Comment by dinger1986** [comment]
Servers aren't configured to restart automatically for obvious reasons, you can schedule a restart by right clicking and go to restart.

**Reply by Alphanumerisch** [reply]
Why is it possible to set this?
And why does the server or workstation not restart even though it is set up?

**Reply by dinger1986** [reply]
Did you set it in your automation policies?

**Reply by Alphanumerisch** [reply]
Yes, it is set up. Restart if required.

**Reply by dinger1986** [reply]
change to Always and see if that helps, if required can be ambiguous

**Reply by Alphanumerisch** [reply]
ok, I will give it a try.
Thanks
